packages/query/tests/query-replication-state.test.ts

```
@@ -41,4 +41,96 @@ describe('QueryReplicationState', () => {
 		await expect(replicationState.nextPage()).resolves.toBeUndefined();
 		expect(runSpy).toHaveBeenCalledTimes(1);
 	});
+
+	it('resumes collection sync when sync-state lookup fails', async () => {
+		const syncStateError = new Error('sync state failed');
+		collectionReplication.syncStateManager.getUnsyncedRemoteIDs.mockRejectedValueOnce(syncStateError);
```

ESLint

[error] Replace `syncStateError` with `⏎↹↹↹syncStateError⏎↹↹`

(prettier/prettier)